src/components/pages/NewParentDashboard/NewParentDashboard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Layout, Card } from 'antd';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+import ProgressCharts from '../ProgressCharts/ProgressCharts';
+
```

Comment:
If we are trying to follow design patterns already established in this project, it is best that this file be renamed to `RenderNewParentDashboard.js`

Reply:
I made the change from NewParentDashboard to RenderNewParentDashboard